### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by sungo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,66 @@
...
-
-## Linux install
+#### Needed Packages
 
-```
-# apt-get install -y build-essential git carton perl-doc postgresql-server-dev-9.5 \
-   postgresql-client-common postgresql-client-9.5
+* PostgreSQL 9.6
+* Git
+* Perl, 5.26.2 or above
```
Comment (karenetheridge) [inline README.md]: pkgsrc perl is only 5.26.1, so I'd just say "5.26", which is what our code asserts.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,66 @@
...
+all devices.
+
+Conch is open source, licensed under MPL2.
+
+## Caveat Emptor
+
+At the time of writing, the API is not considered to be stable. While we do our
+best to prevent breakage, the core is in considerable flux and we do not
+guarantee fit or function right now. The [conch
+shell](https://github.com/joyent/conch-shell) is our current stable interface.
```
Comment (karenetheridge) [inline README.md]: Should we mention the ui as well as a stable interface on top?

Comment (sungo) [reply README.md]: paging @perigrin

Comment (perigrin) [reply README.md]: the shell is our current stable interface.

File: README.md (hunk 1 above)
Comment (sungo) [reply README.md]: done

**Pull request merged**